html/webappapis/dynamic-markup-insertion/opening-the-input-stream/beforeunload.window.js

```diff
@@ -0,0 +1,12 @@
+async_test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => frame.remove());
```

Comment:
This doesn't help much when there's a single test, and leaving it in might make debugging easier.

Reply:
We don't want the test to time out, and having `frame.remove()` here is the easiest way to ensure that the `load` event is fired on the top-level `Window` no matter the outcome of the test.

Reply:
Ah, I assumed it was copypasta from something with multiple tests. Never mind me.